{
	"id": "CVE-2021-3737",
	"description": "A flaw was found in python. An improperly handled HTTP response in the HTTP client code of python may allow a remote attacker, who controls the HTTP server, to make the client script enter an infinite loop, consuming CPU time. The highest threat from this vulnerability is to system availability.",
	"scope": "local",
	"packages": [
		{
			"name": "python2.7",
			"status": "open",
			"nodsa": "Unsupported in Bullseye, only included to build a few applications",
			"nodsa_reason": "ignored",
			"urgency": "not yet assigned",
			"repository": [
				{
					"name": "bullseye",
					"version": "2.7.18-8+deb11u1"
				}
			]
		},
		{
			"name": "pypy3",
			"status": "open",
			"nodsa": "Minor issue",
			"urgency": "not yet assigned",
			"repository": [
				{
					"name": "bullseye-security",
					"version": "7.3.5+dfsg-2+deb11u2"
				},
				{
					"name": "bullseye",
					"version": "7.3.5+dfsg-2+deb11u2"
				}
			]
		},
		{
			"name": "python3.9",
			"status": "open",
			"nodsa": "Minor issue",
			"urgency": "not yet assigned",
			"repository": [
				{
					"name": "bullseye",
					"version": "3.9.2-1"
				}
			]
		}
	]
}
